docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction. Click to **Transaction Detail** page.
+- **(Sending) Address** - The sending address of the transaction. Click the address receiving asset (link) to **Address Detail** page.
+- **Amount of tZEN** - The amount of tZEN transferred in the transaction.
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+## Forward Transfer
+
+When a user requests for tZEN from the Horizen Faucet, the forward transfer transaction is initiated. A forward transfer moves assets from the mainchain to the sidechain. These transactions, more specifically the transaction outputs, are unspendable on the mainchain, but include some metadata so they are redeemable on EON (EVM sidechain). 
+It is the responsibility of EON nodes to monitor the mainchain for incoming transactions, and include them in EON.
+
+The Gobi Testnet Explorer highlights the Forward Transfer transaction:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-forward.png").default} alt="forward-transaction" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **(Receiving) Address** - The recipient address of the transaction.
+- **Amount of tZEN** - The amount of tZEN transferred in the transaction.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+## Fee Payment
+
+The blockchain fee is tZEN-specific fee that is charged to users when processing transactions. The fee is required to process the transaction on the Gobi Testnet and ensure that transfers arrive at their expected destination. The fees are aggregated and paid to forger nodes that have forged at least one block during the last withdrawal epoch.
+
+The Gobi Testnet Explorer highlights the Fee Payment transactions:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-fee.png").default} alt="fee-payment" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Address** - The wallet address receiving the fee payment. 
+- **Amount of tZEN** - The amount of tZEN transferred in the transaction.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+# Blockchain Menu
+
+The Blockchain menu has two feature sections:
+
+- Blocks
+- Transactions
+
+## Blocks 
+The Blocks section includes:
+
+- Blocks

```

Comment:
I'm assuming this is its own separate section inside of the Blocks section? Should there not be a subsection or definition here for `Blocks`?

Reply:
made correction